src/entity/Unit.ts

```diff
@@ -177,6 +180,7 @@ export function create(
       staminaMax,
       attackRange: config.UNIT_BASE_RANGE,
       isMiniboss: false,
+      isPrediction: false,
```

Comment:
undefined is falsy so this could remain an optional property and would function identically (except it wouldn't be serialized which would save some bandwidth).  Why was this made explicit?

Reply:
When I tried to use unit.isPrediction in line 143 of soul_shard.ts it gave me an error that it might be undefined:

```ts
function remove(unit: Unit.IUnit, underworld: Underworld) {
  const soulShardModifier = unit.modifiers[soulShardId];
  if (soulShardModifier) {
    removeShardOwner(soulShardModifier.shardOwnerId, underworld, unit.isPrediction); // <--- HERE
  }
}
```

Wait, could I have just used the !! symbol here?🤦‍♂️I'll revert this......